docs/00-introduction.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<p>maybe add how they are currently using Earth Observation in the notes on workshop users?</p><p> </p><p>the first <code>/collection</code> and <code>/search</code> endpoints don't have the same formatting.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/eoapi-fedgeoday25-workshop/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>